tests/e2e_test.go

```diff
@@ -188,3 +188,57 @@ func TestSignRootTargets(t *testing.T) {
 		}
 	}
 }
+
+func TestSnapshotUnvalidatedFails(t *testing.T) {
+	ctx := context.Background()
+	td := t.TempDir()
+
+	testTarget := filepath.Join(td, "foo.txt")
+	targetsConfig := map[string]json.RawMessage{testTarget: nil}
+
+	if err := os.WriteFile(testTarget, []byte("abc"), 0600); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := createTestHsmKey(td); err != nil {
+		t.Fatal(err)
+	}
+
+	snapshotKey := createTestSigner(t)
+	timestampKey := createTestSigner(t)
+
+	// Initialize succeeds.
+	if err := app.InitCmd(ctx, td, "", 1, targetsConfig, snapshotKey, timestampKey); err != nil {
+		t.Fatal(err)
+	}
+
+	// Validate that root and targets have one unfilled signature.
+	// TODO: When #281 is merged, add a test with a valid threshold achieved
```

Comment:
#281 in this repo is a merged PR.

Reply:
yep! it was merged concurrently -- working on resolving that TODO now :)